### PR TITLE
Tweak readme.md

### PR DESCRIPTION
**Pull request by qwertychouskie**
Mainly grammar and consistency fixes.
Also use KiwiIRC instead of "qwebirc".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ To run SuperTuxKart, make sure that your computer's specifications are equal or
 * Ideally, you'll want a joystick with at least 6 buttons.
 
 ## License
-The software is released under the GNU General Public License (GPL) which can be found in the file [`COPYING`](/COPYING) in the same directory as this file. Information about the licenses for the artwork is contained in `data/licenses`.
+The software is released under the GNU General Public License (GPL) which can be found in the file [`COPYING`](/COPYING) in the same directory as this file. Information about the licenses for the artwork is contained in [`data/CREDITS`](data/CREDITS).
```
Comment (auriamg) [inline README.md]: This is wrong, it was correct before. There are no licenses in credits.

File: README.md [hunk 2]
```diff
@@ -73,12 +73,12 @@ Compilation instruction is explained there. If you don't need this feature, pass
 Run the following commands to compile SuperTuxKart:
 
 ```bash
-mkdir cmake_build
-cd cmake_build
+mkdir build
```
Comment (auriamg) [inline README.md]: why would you change that? please stop randomly changing stuff without reason

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # SuperTuxKart
 [![Build Status](https://travis-ci.org/supertuxkart/stk-code.svg?branch=master)](https://travis-ci.org/supertuxkart/stk-code)
 [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/supertuxkart/stk-code?svg=true&branch=master)](https://ci.appveyor.com/project/supertuxkart/stk-code)
-[![#supertuxkart on the freenode IRC network](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://webchat.freenode.net/?channels=supertuxkart)
+[![#supertuxkart on the freenode IRC network](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://kiwiirc.com/client/irc.freenode.net/#supertuxkart)
```
Comment (auriamg) [inline README.md]: why would you change that? we used the official freenode client

File: README.md [hunk 4]
```diff
@@ -189,23 +188,26 @@ You'll have to run these commands inside your stk-code directory.
 With clang:
 
 ```bash
-mkdir cmake_build
-cd cmake_build
+mkdir build
```
Comment (auriamg) [inline README.md]: Why would you change that?

File: README.md (hunk 3 above)
Comment (qwertychouskie) [reply README.md]: It's very hard to keep track of what's being said vs connect/disconnect messages, making it very easy to miss replies, etc.

vs

(Plus having to do a captcha on qwebirc is annoying, KiwiIRC doesn't have a captcha ;)

**Comment by Benau** [comment]
Your PR atm has too many conflicts to merge

**Pull request closed without merging**